src/Services/Form.php

```diff
@@ -230,15 +230,15 @@ public function labels(bool $labels): self
 
     public function sectionVisibility($fields, bool $hidden): self
     {
-        Collection::wrap(is_string($fields) ? func_get_args() : $fields)
+        Collection::wrap(is_string($fields) ? [$fields] : $fields)
             ->each(fn ($field) => $this->section($field)->put('hidden', $hidden));
 
         return $this;
     }
 
     public function tabVisibility($tabs, bool $hidden): self
     {
-        $tabs = Collection::wrap(is_string($tabs) ? func_get_args() : $tabs);
+        $tabs = Collection::wrap(is_string($tabs) ? [$tabs] : $tabs);
 
         $this->template->get('sections')->each(fn ($section) => $tabs->when(
             $tabs->contains($section->get('tab')),
```

Comment:
if a collection contains `true`, the `->contains()` method will always return `true` since that's how php works for some reason ( see above comment and below picture )

<img width="395" alt="image" src="https://github.com/laravel-enso/forms/assets/37445394/2e59d69d-06ae-4448-92ba-361e2379d21d">